content/en/docs/concepts/scheduling-eviction/pod-scheduling-readiness.md

```diff
@@ -91,8 +91,6 @@ scheduling. You can use `scheduler_pending_pods{queue="gated"}` to check the met
 
 ## Mutable Pod Scheduling Directives
```

Comment:
For GA, we like to follow the style guide, so I recommend
```suggestion
## Mutable Pod scheduling directives
```

but, it won't block a merge.

Reply:
Makes sense to me. Updated. PTAL.